_includes/sections/dns.html

```diff
@@ -285,6 +285,22 @@ <h4>Terms</h4>
     <li>DNSCrypt - An older yet robust method of encrypting DNS.</li>
   </ul>
 
+  <h4>How to Validate</h4>
+
+  <ul>
+    <li>DoH / DoT
+      <ul>
+        <li>Check <a href="https://www.dnsleaktest.com/">https://www.dnsleaktest.com/</a>.</li>
+        <li>Check the website of your DNS provider. They may have a page for telling "you are using X DNS." Examples include <a href="https://adguard.com/en/adguard-dns/overview.html">AdGuard</a> and <a href="https://1.1.1.1/help">Cloudflare</a>.</li>
+        <li>If using Firefox's trusted recursive resolver (TRR), navigate to <code>about:networking#dns</code>. If the TRR column says "true" for some fields, you are using DoH.</li>
```

Comment:
```suggestion
        <li>If using Firefox's trusted recursive resolver (TRR), navigate to <code>about:networking#dns</code>. If the TRR column says "true", then that DNS query was done using DoH.</li>
```

Can we assume that people read our insturctions and we don't need to separately warn about not all queries possibly being TRR?

![image](https://user-images.githubusercontent.com/831184/63222112-a8a0b080-c192-11e9-98b5-64bcfeb6a360.png)

Reply:
Hmm... that's a good point... there's ongoing discussions about how the about:config tweaks section could potentially go or not so maybe its worth calling out in a warning or something here about this?

Reply:
I'll push an update.